myst_nbs/time_series/Forecasting_with_structural_timeseries.myst.md

```diff
@@ -0,0 +1,662 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.7
+kernelspec:
+  display_name: Python 3.9.6 ('prod_streamlit')
+  language: python
+  name: python3
+---
+
+(forecasting_with_ar)=
+# Forecasting with Structural AR Timeseries
+
+:::{post} Oct 20, 2022
+:tags: forecasting, autoregressive, bayesian structural timeseries
+:category: intermediate
+:author: Nathaniel Forde
+:::
+
++++
+
+Bayesian structural timeseries models are an interesting way to learn about the structure inherent in any observed timeseries data. It also gives us the ability to project forward the implied predictive distribution granting us another view on forecasting problems. We can treat the learned characteristics of the timeseries data observed to-date as informative about the structure of the unrealised future state of the same measure. 
+
+In this notebook we'll see how to fit an predict a range of auto-regressive structural timeseries models and importantly how to predict future observations of the models.
```

Comment:
```suggestion
In this notebook we'll see how to fit a range of auto-regressive structural timeseries models and importantly how to predict future observations of the models.
```

Reply:
Ah! I think I get you. Will review the links later tonight and try and adjust to the prediction step everywhere. Any help would be much appreciated! Will add any questions here I find in testing!